### ISSUE TITLE
Less dependencies

### ISSUE THREAD

**Issue by Leont** [Perl-Toolchain-Gang/Software-License#38]
I've recently started using Software::License in App::ModuleBuildTiny. However, it almost doubled my list of dependencies (15 to 26). Reducing its weight would be most welcome. Data::Section in particular feels rather heavy (and I suspect the direct dependency on Sub::Install isn't necessary either).

**Comment by dolmen** [comment]
:+1:

**Comment by karenetheridge** [comment]
:+1:

**Comment by neilb** [comment]
:+1:

**Comment by miyagawa** [comment]
Sub::Install is dropped already https://github.com/Perl-Toolchain-Gang/Software-License/commit/397c16bd717d7af0dfc0cf58ee3a41585117b71e and it just needs a release.

**Comment by rjbs** [comment]
The ayes have it.  I have made an immediate release.

**Comment by Leont** [comment]
> The ayes have it. I have made an immediate release.

I'm confused now, the issue is closed but the main issue hasn't really changed.

**Comment by miyagawa** [comment]
Yeah I didn't mean to suggest that this is fixed - another beast is Data::Section.

**Comment by rjbs** [comment]
Re-opened!  I lost the original point by only reading as many comments as were still in my mailbox.

**Comment by rjbs** [comment]
BTW, I am happy to see this addressed by a PR.
